examples/david_map_reduce_weighted.py

```diff
@@ -208,7 +227,9 @@ def main(
         -herding_coreset.coreset.data[:, 0],
         c=herding_coreset.coreset.data[:, 2],
         cmap="gray",
-        s=np.exp(2.0 * coreset_size * herding_weights).reshape(1, -1),
+        s=(5.0 * coreset_size * herding_weights * downsampling_factor**2).reshape(
```

Comment:
This works, but the marker size appears very uniform - this could just be because the weights _are_ quite uniform, though! (They're all in the range `(0.0070, 0.0082)` when I run with `downsampling_factor=8`.) Is it worth fiddling with this to get more of a visual size difference?